src/markdown-pages/landing/landing-es.md

```diff
@@ -2,31 +2,31 @@
 lang: es
 templateKey: landing-page
 slug: home
-pageTitle: es Project | Home
+pageTitle: Proyecto | Inicio
 hero:
-  heading: es Customizable Landing Page Template 
-  text: es Set up a landing page for your agency with this template. It includes all of the resources that you need to have an easy, compliant, secure, appealing, and sustainable landing page.
+  heading: Plantilla Editable para Página de Inicio 
+  text: Crea una página de inicio para tu institución de gobierno con esta plantilla. Incluye todos los recursos que puedes necesitar para tener una página de inicio simple, segura, atractiva y sostenible. 
   buttonLink: https://github.com/nyc-cto/LPaaS2/
```

Comment:
`buttonLink` for github and contact can be moved to links constants

Reply:
Hero and section button links moved to `src/constants/links.js`.